storage/spring-cloud-azure-starter-storage-blob/storage-blob-sample/README.md

```diff
@@ -120,7 +119,7 @@ $ curl http://localhost:8080/blob -d "new message" -H "Content-Type: text/plain"
 Verify by sending a GET request
 
 ```shell
-$ curl -XGET http://localhost:8080/blob
+curl -XGET http://localhost:8080/blob
```

Comment:
The same for line #116 $ curl http://localhost:8080/blob -d "new message" -H "Content-Type: text/plain"